command_line/background.py

```diff
@@ -127,12 +128,7 @@ def run(args=None):
                     label = f"image {indx}" if len(images) > 1 else ""
                 ds2 = 1 / flex.pow2(d)
                 ax.plot(ds2, I, label=label)
-            xticks = ax.get_xticks().tolist()
-            ax.xaxis.set_major_locator(mticker.FixedLocator(xticks))
-            x_tick_labs = [
-                "" if e <= 0.0 else f"{math.sqrt(1.0 / e):.2f}" for e in xticks
-            ]
-            ax.set_xticklabels(x_tick_labs)
+                ax.xaxis.set_major_formatter(resolution_formatter)
```

Comment:
I think the original indentation was correct.  We are putting all these plots on the same axes, so we only need to set the tick format once.
```suggestion
            ax.xaxis.set_major_formatter(resolution_formatter)
```